component/service/src/test/java/org/exoplatform/social/service/rest/IntranetNotificationsRestServiceTest.java

```
@@ -133,6 +133,22 @@ public void testIgnoreInvitationToJoinSpace() throws Exception {
     
     spaceService.deleteSpace(space.getId());
   }
+
+  public void testUnauthorizedUserRequestValidation() throws Exception {
+      Space space = getSpaceInstance(1);
+
+      end();
+      begin();
+
+      startSessionAs("john");
```

It should be mary no ?

---

mary is the account used to test that see make a real request. 
The validator of the request have no impact.

---

Actually we are in the case where the space manager wants to accept an space invitation for an user whereas this user has not been invited, right ? So it is not a permission issue, but just an operation which is not allowed (no matter what the user permissions are), so it should return a 403 error IMO. WDYT ?